src/components/monthly-upgrade.js

```diff
@@ -41,6 +46,12 @@ var MonthlyUpgrade = React.createClass({
       return;
     }
 
+    reactGA.event({
```

Comment:
This fires after the submission goes through completely?

Reply:
That's based on the check for `state.submitting`...I'm trying to think through if there's any opportunities for this to give us bad data.

Reply:
No, on the button click.

Reply:
I think, let me double check. On click was my intent.

Reply:
Yeah, it's onClick